### DISCUSSION TITLE
HIPAA compliance

### DISCUSSION THREAD

**Discussion by jaredgibb** [supabase/supabase#1219] [Questions] [answered]
Hey guys, i see on your TOS that you will sign a BAA. I just want to make sure i understand this correctly but that would mean i can use your service for auth and data storage that is hipaa compliant?!

**Comment by kiwicopple** [comment]
HI @jaredgibb - we are not yet HIPAA compliant, but this is definitely one of our goals. We are working towards SOC2 first.

Of course you can also self-host too (in which case you will be responsible for the compliace/security).

**Reply by ClaudiuBadau** [reply]
@kiwicopple , just wondering any timeline on SOC2?

**Reply by jaredgibb** [reply]
> HI @jaredgibb - we are not yet HIPAA compliant, but this is definitely one of our goals. We are working towards SOC2 first.
> 
> Of course you can also self-host too (in which case you will be responsible for the compliace/security).

The reason I ask is that in your TOS you speak of signing a BAA. I am hoping to see that soon. You guys still kill it if you can make it happen!!!

**Reply by kiwicopple** [reply]
> just wondering any timeline on SOC2?

We're now SOC2 compliant 🎉

**Reply by SamMakesThings** [reply]
Hey @kiwicopple - any updates on the HIPAA compliance? Evaluating Supabase for a new project and need to know if it is a good option yet. 🙂

**Reply by JanRuettinger** [reply]
Any updates on HIPPa compliance?

**Reply by ThatGuySam** [reply]
Looks like Supabase communicates on their site they are now HIPAA and SOC2 Compliant: 
https://supabase.com/security

> Supabase is HIPAA compliant. Enterprise and Teams customers can request to sign our BAA [here](https://forms.supabase.com/hipaa).

**Comment by ThatGuySam** [comment, marked as answer]
It would seem that the answer is now yes: 

> Looks like Supabase communicates on their site they are now HIPAA and SOC2 Compliant: https://supabase.com/security
> 
> > Supabase is HIPAA compliant. Enterprise and Teams customers can request to sign our BAA [here](https://forms.supabase.com/hipaa).
